packages/cli/src/commands/dids.ts

```diff
@@ -0,0 +1,29 @@
+import {Command} from 'commander';
+import inquirer from 'inquirer';
+import axios from 'axios';
+import {IWallet} from '@docknetwork/wallet-sdk-core/lib/types';
+import {getWallet} from '../helpers';
+
+const didsCommand = new Command('dids').description('list, create');
+
+didsCommand
+  .command('list')
+  .description('List wallet dids')
+  .action(async () => {
+    const wallet: IWallet = await getWallet();
+
+    const dids = await wallet.getDocumentsByType('DIDResolution');
+
+    console.log(dids);
+  });
+
+didsCommand
+  .command('create')
+  .description('List wallet dids')
+  .action(async () => {
+    const wallet: IWallet = await getWallet();
+
+    // implement did creation logic, start with did key
```

Comment:
are you tracking all these improvement thoughts anywhere else? I feel like these comments might get lost

Reply:
Yes. I pushed this as placeholder for only. But I'm currently working on migrating the basic verification flow to the wallet-sdk. DID creation is a requirements on it. And I will leave wallet-backups (import, export), accounts, and transactions for the future.